embedded-workshop-book/src/uarte-implementation.md

```diff
@@ -0,0 +1,183 @@
+# Write the Uarte implementation
+## Step-by-Step Solution
+
+### Step 1: Check Documentation.
+
+The UART protocol requires four pins, they are usually labelled:
+* RXD
+* TXD
+* CTS
+* RTS
+
+✅ Check the User Guide in section 7.2 to find to find out which pins are reserved for these and what their configuration needs to be.  
```

Comment:
We should link to the nRF52840 User Guide here (and make it clear we mean that user guide, as it's the first mention on this page and you might arrive without context). 

However, the HTML version of the user guide doesn't have chapter numbers (!!). So we should either:

* Link to the PDF and quote the chapter number (7.2)
* Link to the start of the HTML and give the chapter path ([nRF52 Series](https://infocenter.nordicsemi.com/topic/struct_nrf52/struct/nrf52.html) > [nRF52840](https://infocenter.nordicsemi.com/topic/struct_nrf52/struct/nrf52840.html) > [nRF52840 DK ](https://infocenter.nordicsemi.com/topic/ug_nrf52840_dk/UG/dk/intro.html)> [Interface MCU](https://infocenter.nordicsemi.com/topic/ug_nrf52840_dk/UG/dk/if_mcu.html) > [Virtual COM port](https://infocenter.nordicsemi.com/topic/ug_nrf52840_dk/UG/dk/vir_com_port.html))
* Link to the correct HTML page (<https://infocenter.nordicsemi.com/topic/ug_nrf52840_dk/UG/dk/vir_com_port.html>)